tutorials/scripting/gdscript/warning_system.rst

```diff
@@ -23,14 +23,20 @@ script editor's status bar. The example below has 2 warnings:
 
 .. image:: img/typed_gdscript_warning_example.webp
 
-To ignore specific warnings in one file, insert an annotation of the
-form ``@warning_ignore("warning-id")``, or click on the ignore link to the
-left of the warning's description. Godot will add an annotation above the
-corresponding line and the code won't trigger the corresponding warning
-anymore:
+To ignore single warnings within a file, use the
+:ref:`@warning_ignore <class_@GDScript_annotation_@warning_ignore>` annotation.
+You can click on the ignore link to the left of the warning's description.
+Godot will add an annotation above the corresponding line and the code
+won't trigger the corresponding warning anymore:
 
 .. image:: img/typed_gdscript_warning_system_ignore.webp
 
+To ignore multiple warnings in a region within a file, use the
+:ref:`@warning_ignore_start <class_@GDScript_annotation_@warning_ignore_start>`
+and :ref:`@warning_ignore_restore <class_@GDScript_annotation_@warning_ignore_restore>`
+annotations. You can omit the latter if you want to ignore the specified warning types
+until the end of the file.
```

Comment:
```suggestion
annotations. You can omit ``@warning_ignore_restore`` if you want to ignore the
specified warning types until the end of the file.
```
Optional. It's in the writing guidelines to avoid "former" and "latter". In this case I think it improves things somewhat.